sections/04_study.md

```diff
@@ -185,9 +185,71 @@ disease, which will lead to novel diagnostics and therapeutics.
 
 ### Transcription factors and RNA-binding proteins
 
-*Existing reviews have covered some of these papers rather well and we do not
-want to repeat what has already been well-stated elsewhere.  This could
-be split into two sub-sections or kept very brief.*
+Transcription Factor and RNA-binding proteins are key components for gene
+regulation, making them very important to understand for higher level
+biological processes. While high-throughput sequencing techniques such as
+chromatin immunoprecipitation and massively parallel DNA
+sequencing (ChIP-seq) have been able to accurately identify binding regions
+for DNA and RNA proteins, these experiments are both time consuming and
+expensive. In addition, the sequencing methods do not provide any sort of
+analysis on the proteins which would lead to a better understanding of
+the underlying process. Thus, there is a need to computationally predict
+and understand these binding regions de novo from sequences.
+
+#### Transcription Factors
+
+Transcription Factors (TFs) are regulatory proteins that bind to certain
+locations on a DNA sequence and control the rate of mRNA
+production. ChIP-seq and related technologies are able to identify highly likely binding
+sites for a certain TF, and databases such as ENCODE have provided ChIP-seq
+data for hundreds of different TFs across many laboratories. However, ChIP-seq
+experiments are expensive and time consuming. Since the data that scientists
+have discovered is available, in silico methods to predict binding sites are of
+great interest, thus eliminating the need to do new ChIP-seq experiments every
+time analysis is done on a new sequence.
+
+In order to computationally predict TFBSs on a DNA sequence, researchers
+initially used consensus sequences and position weight matrices to match
+against a test sequence [@tag:Stormo2000_dna]. Simple neural network
+classifiers were then proposed to differentiate positive and negative binding
+sites, but did not show significant improvements over the weight matrix
+matching methods [@tag:Horton1992_assessment]. Later, SVM techniques
+outperformed the generative methods by using k-mer features
+[@tag:Ghandi2014_enhanced @tag:Setty2015_seqgl], but string kernel based SVM
+systems are limited by expensive computational cost proportional to the number
+of training and testing sequences. More recently,
+[@tag:Alipanahi2015_predicting] showed that convolutional neural network
```

Comment:
I think it is okay to not describe all of the work that has followed DeepBind, in part because of the good existing reviews, but we should make it more obvious that there is _a lot_ of other work in this area in the past couple years.  Our goal is not to enumerate all of it, but rather to say conclude something about the successes, limitations, challenges, opportunities, etc. for deep learning in this area (as is done below with visualization).

I'd like to merge soon, so maybe this should be a TODO instead of trying to make the revisions in this first draft.